### PR TITLE
Dm/add ahoy events

### PR DESCRIPTION
**Pull request by JoshingYou1**
## Description - what does this code do?
Adds event tracking for user role and practice comments.

## Testing done - how did you test it/steps on how can another person can test it 
1. Log in as any type of user.
2. Go to the server logs in your rails server and look for an `Ahoy::Event` log with the name `"User role"`. Make sure it contains your `username` and `user_role`.
3. Go to any practice's show view and create a comment.
4. Go to the server logs just like above, but look for an event called `"Practice comment created"`. Make sure it contains the `comment_id`, `creator_id`, `editor_id`, and `body`.

## Screenshots, Gifs, Videos from application (if applicable)
Untitled
Screen Shot 2020-10-26 at 1 57 25 PM

## Acceptance criteria
- [ ]

## Definition of done
- [ ] Unit tests written (if applicable)
- [ ] e2e/accessibility tests written (if applicable)
- [X] Events are logged appropriately
- [ ] Documentation has been updated, if applicable
- [ ] A link has been provided to the originating JIRA issue
- [X] No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs

### CODE REVIEW

File: db/schema.rb [hunk 1]
```diff
@@ -10,7 +10,7 @@
 #
```
Comment (emyl3) [inline db/schema.rb:10]: we don't want these file changes in `master`.
let me know if you want to reset this file to match `master` or you want me to do it! :)

**Review by emyl3 — CHANGES_REQUESTED** [review]
thanks for remembering this, josh! 😅

**Review by emyl3 — APPROVED** [review]
Thank you, josh!

**Review by bradjohnson92008 — APPROVED** [review]
I see the events in the log.  Thx Josh.

**Pull request merged**